exasol_script_languages_developer_sandbox/cli/commands/setup_ec2.py

```diff
@@ -1,29 +1,29 @@
 from typing import Optional
 
-import click
-
 from exasol_script_languages_developer_sandbox.cli.cli import cli
 from exasol_script_languages_developer_sandbox.cli.common import add_options
 from exasol_script_languages_developer_sandbox.cli.options.aws_options import aws_options
+from exasol_script_languages_developer_sandbox.cli.options.ec2_options import ec2_key_options
+from exasol_script_languages_developer_sandbox.cli.options.id_options import id_options
 from exasol_script_languages_developer_sandbox.cli.options.logging import logging_options, set_log_level
+from exasol_script_languages_developer_sandbox.lib.asset_id import AssetId
 from exasol_script_languages_developer_sandbox.lib.aws_access import AwsAccess
 from exasol_script_languages_developer_sandbox.lib.run_setup_ec2 import run_setup_ec2
 
 
 @cli.command()
 @add_options(aws_options)
 @add_options(logging_options)
-@click.option('--ec2-key-file', required=False, type=click.Path(exists=True, file_okay=True, dir_okay=False),
-                 default=None, help="The EC2 key-pair-file to use. If not given a temporary key-pair-file will be created.")
-@click.option('--ec2-key-name', required=False, type=str,
-             default=None, help="The EC2 key-pair-name to use. Only needs to be set together with ec2-key-file.")
+@add_options(ec2_key_options)
+@add_options(id_options)
 def setup_ec2(
             aws_profile: str,
             ec2_key_file: Optional[str],
             ec2_key_name: Optional[str],
+            asset_id: str,
             log_level: str):
     """
     Debug command to test setup of an EC-2 instance.
     """
     set_log_level(log_level)
-    run_setup_ec2(AwsAccess(aws_profile), ec2_key_file, ec2_key_name)
+    run_setup_ec2(AwsAccess(aws_profile), ec2_key_file, ec2_key_name, AssetId(asset_id))
```

Comment:
good that you introduced a class for the AssetId, but what is here meant with it. The id we tag all resources with?

Reply:
Under normal circumastances (release-flow!!!) the asset-id will contain the version of the slc package (e.g. `5.0.0`)
The class is used for (currently) 3 things:
- The tag value used to identify later all resources: cloudformation-stack (and it's belonging resources: ec2-instance and security-group), key-pair, snapshot, ami, export-image-tasks
- The name of the AMI
- The prefix in the s3-bucket for the vm-image-files (helps to separate the images-files by folder, for example there will be separate folders in the s3 bucket for 5.0.0, 6.0.0, and so on....)

Actually I am not so happy with the name. I was thinking a while about a better naming, "global-resource-id" and "id-namespace" came to my mind, but also was not much better. Maybe you have a good idea.